gulpfile.js

```diff
@@ -9,7 +9,7 @@ var gulp = require('gulp'),
 gulp.task('build-theme', function () {
     return gulp.src(['scss/bootstrap-dark.scss'])
         .pipe(sourcemaps.init())
-        .pipe(sass().on('error', sass.logError))
+        .pipe(sass({ includePaths: [__dirname] }).on('error', sass.logError))
```

Comment:
Yes

It was needed because I included this stylesheet directly into my project and parcel does exactly the same thing as the includePaths statement, so I could work arround the issue of the bootstrap package location being somewhere else